### ISSUE TITLE
Browser's don't necessarily represent users

### ISSUE THREAD

**Issue by richsalz** [intarchboard/for-the-users#1]
I have several concerns about this paragraph:

```
For example, one of the most successful Internet applications is the Web. One of its key implementation roles is that of the Web browser -- called the User Agent in {{?RFC7230}} and other specifications. Because there is more than one implementation of the standards that specify a Web browser, there is a natural competition between them to do carefully consider the user's needs as an agent. As a result, Web browsers' interests are better aligned with those of their users, creating an ecosystem that is positively user-focused.
```
First, the minor one is using the term "User Agent" to help justfiy the claim; that should be removed.  UA was picked because it is a term of art of an end-user facing program, contrasted with something like an SMTP or IMAP3 server.  (I picked those deliberately, as UA was first coined for SMPT IIIRC.)

My fundamental concern, however, is the claim that there is natural competition, and that this leads to user alignment. First, the market is wiped out natural competition -- one browser seems to be nearly 2/3 of the market by most measurements. Since it is forked into several other browsers, an estimate of almost 80% seems reasonable.

If there is no competition, how can there be natural alignment? Compare things like tracker and ad-blocking from the top three browers.  If a browser is distributed for free by an ad company, how confident can we be that user privacy -- something the IETF continues to emphasize -- is a goal?

**Comment by letoams** [comment]
While I think that the usage here is correct, I do agree for those people not very familiar with User Agent, bringing this up is just a distraction in this paragraph. Just stating that webbrowsers must act in the best interest of their end user says the same thing, without needing to refer to some very specific HTTP header.

**Comment by bhstark2** [comment]
I completely disagree with the entire premise of this section, or that there exists some sort of natural pressure for web browsers to act in the best interest of users. Because web browsers have the most control over what is presented to eyeballs, gaining control over them and the data they have about their end users is critical to many companies' business models. The biggest pressure I see is legal/regulatory -- which is true of all parts of the ecosystem.
Some things that suggest the premise is incorrect include: widespread malware (Chromium and IE) that users are "encouraged" to download that do DNS hijacking and other nefarious actions; Lenovo laptop "support" for Superfish adware; enterprises that customize browsers in ways to ensure users cannot deleted browser history; Chrome privacy policy that tells users their browsing history is used to create a profile of them (for advertising) if they're logged in. I could come up with more examples.
I'm curious what proof exists that suggests competition creates a natural tendency towards respect of user privacy?
